test/libsolidity/syntaxTests/nameAndTypeResolution/363_non_payable_constructor.sol

```diff
@@ -8,4 +8,4 @@ contract D {
     }
 }
 // ----
-// TypeError: (106-119): Member "value" not found or not visible after argument-dependent lookup in function () returns (contract C) - did you forget the "payable" modifier?
+// TypeError: (106-119): Member "value" is only available for external payable function types
```

Comment:
We could be more precise here. The member would be available if the constructor of `C` was payable.

Reply:
Any suggestion on the exact phrasing? Maybe

```
Member \"value\" is only available for external payable constructors. 
```

Reply:
Perhaps `Member \"value\" is only available if the constructor of "<contractName>" is payable.`